### ISSUE TITLE
Banner label link contrast failure on focus

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-lively#6]
With focus, linked labels within configured banners have only a 1.7:1 contrast, well under the 4.5:1 minimum. The text becomes `rgb(10, 79, 158)` on the `rgb(214, 45, 106)` background.

**Comment by nelsonamaya82** [comment]
Thanks @aviannamiller, this is fixed and merged into the `main` branch.
